src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -1379,6 +1391,14 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
         }
     }
 
+    @computed get isPercentage() {
+        return this.discreteVsDiscretePlotType === DiscreteVsDiscretePlotType.PercentageStackedBar;
+    }
+
+    @computed get isStacked() {
+        return this.discreteVsDiscretePlotType === DiscreteVsDiscretePlotType.StackedBar || this.discreteVsDiscretePlotType === DiscreteVsDiscretePlotType.PercentageStackedBar;
+    }
+
```

Comment:
I was drawn to this because I think the names are confusing because they specifically refer to stacked bar plots, but they're in the general plots tab component. But on reading closely I think they should both just be consts declared in the plot() getter, in the MultipleCategoryBarPlot part, since they're only used in that one place.

Reply:
updated